test/IntegrationTests/SmokeTests.cs

```diff
@@ -89,6 +89,28 @@ public void ApplicationIsExcluded()
         VerifyTestApplicationNotInstrumented();
     }
 
+#if NET6_0_OR_GREATER
+    [Fact]
+    [Trait("Category", "EndToEnd")]
+    public void ApplicationFailFast()
+    {
+        SetEnvironmentVariable("OTEL_DOTNET_AUTO_EXCLUDE_PROCESSES", $"dotnet,dotnet.exe,{EnvironmentHelper.FullTestApplicationName},{EnvironmentHelper.FullTestApplicationName}.exe");
+        SetEnvironmentVariable("OTEL_DOTNET_AUTO_FAIL_FAST_ENABLED", "true");
```

Comment:
The test is showing a problem: if some processes are excluded one would expect them to keep running without instrumentations, but, with the fail fast these processes can't be started. At first it seems that the process exclusion shouldn't be a rule like the others.

Reply:
I agree this probably won't be a frequent/common case. However, it is a (small) bug. I'm fine with fixing it separately, but, likely something worth fixing before GA.

Reply:
I think it is feature (FAIL_FAST takes precedence over another settings) not a bug.
I documented behavior.
Please let me know if you are fine with this 83c684f7818bd480241c24fd8954005a4b4a6e6a